test/hummingbot/connector/exchange/ascend_ex/test_ascend_ex_user_stream_tracker.py

```diff
@@ -47,10 +46,10 @@ def _authentication_response(self, authenticated: bool) -> Dict[str, Any]:
         return message
 
     @aioresponses()
-    @patch("aiohttp.client.ClientSession.ws_connect")
+    @patch("aiohttp.client.ClientSession.ws_connect", new_callable=AsyncMock)
     def test_listen_for_user_stream_authenticates_and_subscribes_to_events(self, api_mock, ws_connect_mock):
         output_queue = asyncio.Queue()
-        self.ev_loop.create_task(self.tracker.data_source.listen_for_user_stream(self.ev_loop, output_queue))
+        task = self.ev_loop.create_task(self.tracker.data_source.listen_for_user_stream(self.ev_loop, output_queue))
```

Comment:
We should store the task in `self.listening_task` since that is cleaned up during `tearDown` processing